server/controller.js

```diff
@@ -355,20 +355,21 @@ var fs = require('fs-extra'),
 			return parseAfter(parseMain(parseBefore(s)));
 		};
 	};
-	var window = require("jsdom-nogyp").jsdom(null, null, {features: {FetchExternalResources: false}}).parentWindow;
+	var jsdom = require("jsdom-nogyp");
 	var htmlMd = require('html-md-optional_window');
-	// using my fork of html-md, we create the window via jsdom once at the top, then just pass the reference,
-	// which will avoid jsdom.jsdom().createWindow() every time, much, much faster, and avoids memory leaks
-	Controller['html-to-md'] = (function(window){
+	Controller['html-to-md'] = function(){
 		var brRe = /<br\s*(\/)?>/gmi;
 		var entities = new (require('html-entities')).AllHtmlEntities();
 		return function(str){
+			var window = jsdom.jsdom(null, null, {features: {FetchExternalResources: false}}).parentWindow;
 			str = str || '';
 			str = entities.decode(str);
 			str = htmlMd(str, {window: window}).replace(brRe, "\n");
```

Comment:
if you don't want use the optional window, you don't have to, but that was causing memory issues for me in the past.
you can just do this instead

```
    str = htmlMd(str).replace(brRe, "\n");		// then you don't need to require jsdom anymore.
```